src/mouse.cpp

```diff
@@ -589,9 +595,13 @@ void GraphicsWindow::MouseRightUp(double x, double y) {
                 if(r->type == Request::Type::CUBIC) addAfterPoint++;
                 AddContextMenuItem("Add Spline Point", ContextCommand::ADD_SPLINE_PT);
             }
+
         }
 
         if(gs.points == 1) {
+			
+			AddContextMenuItem("Fillet", ContextCommand::TANGENT_ARC);
```

Comment:
This should be called something like "Create Tangent Arc"; we don't call it "Fillet" elsewhere.